app/src/main/res/layout-sw600dp-land/topic_lessons_story_summary.xml

```diff
@@ -103,6 +105,7 @@
 
               <TextView
                 android:id="@+id/story_name_text_view"
+                style="@style/TextViewStart"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
```

Comment:
I think we can remove `layout_width` and `layout_height` from all files if they are `wrap_content`

Reply:
No, actually we decided to keep it as it is for now. As this PR only focus on getting text alignment in RTL. This was also mentioned in one of the comments in my PR [#3989](https://github.com/oppia/oppia-android/pull/3989). Otherwise the PR might go in much detail for all the TextViews. There is going to be separate issue and PR that will revise the styles and its attributes.

Reply:
Ack.